agent/main.go

```diff
@@ -275,3 +283,15 @@ func pipeTapChannelToSocket(connection *websocket.Conn, messageDataChannel <-cha
 		}
 	}
 }
+
+func getSyncEntriesConfig() *shared.SyncEntriesConfig {
+	var syncEntriesConfig = &shared.SyncEntriesConfig{}
+
+	syncEntriesConfigJson := os.Getenv(shared.SyncEntriesConfigEnvVar)
+	err := json.Unmarshal([]byte(syncEntriesConfigJson), syncEntriesConfig)
+	if err != nil {
+		panic(fmt.Sprintf("env var %s's value of %s is invalid! json must match the shared.SyncEntriesConfig struct, err: %v", shared.SyncEntriesConfigEnvVar, syncEntriesConfigJson, err))
```

Comment:
it feels that in this case we should "log.exception" - which prints the exception to log and panic